ts/private/ts_project.bzl

```diff
@@ -259,23 +270,25 @@ See https://github.com/aspect-build/rules_ts/issues/361 for more details.
         typecheck_outs.extend(output_types)
 
     if use_tsc_for_js or use_tsc_for_dts:
-        tsc_emit_arguments = ctx.actions.args()
+        arguments = ctx.actions.args()
+        for mod in argument_mods:
+            mod(arguments)
 
         # Type-checking is done async as a separate action and can be skipped.
         if ctx.attr.isolated_typecheck:
-            tsc_emit_arguments.add("--noCheck")
-            tsc_emit_arguments.add("--skipLibCheck")
-            tsc_emit_arguments.add("--noResolve")
+            arguments.add("--noCheck")
+            arguments.add("--skipLibCheck")
+            arguments.add("--noResolve")
 
         if not use_tsc_for_js:
             # Not emitting js
-            tsc_emit_arguments.add("--emitDeclarationOnly")
+            arguments.add("--emitDeclarationOnly")
 
         elif not use_tsc_for_dts:
             # Not emitting declarations
             # TODO: why doesn't this work with workers?
```

Comment:
Does use of a single `ctx.actions.args()` fix this?

Reply:
Works on my machine :)